### DISCUSSION TITLE
This computer will be shut down shortly

### DISCUSSION THREAD

**Discussion by rappc87** [AlexGustafsson/homebridge-wol#263] [Questions]
### Issue

net rpc shutdown --ipaddress 192.168.1.x --user username%password after the command this computer will be shut down shortly and after about 3-4 seconds the computer shuts down. i want it to shut down directly without this warning. what should i change the command to?

### Environment

```Markdown
.
```

### Logs

```Markdown
.
```

**Comment by AlexGustafsson** [comment]
Try to add the `-t 0` flag. See: https://stackoverflow.com/questions/9936986/shutdown-windows-machine-from-linux-terminal.
